lib/next_rails/bundle_report.rb

```diff
@@ -58,6 +59,45 @@ def self.gem_header(_gem)
       header
     end
 
+    def self.compatible_ruby_version(rails_version)
+      # find all the versions of rails gem
+      uri = URI('https://rubygems.org/api/v1/versions/rails.json')
+      res = Net::HTTP.get_response(uri)
+      all_versions_res = JSON.parse(res.body)
+
+      # push all the versions in an array
+      all_versions = []
+      all_versions_res.each { |rv| all_versions << rv['number'] }
+
+      rv = rails_version[:rails_version]
+      matched_versions = all_versions.select { |h| h.match(rv) && h.start_with?(rv) }
+
+      # the list can either have the exact version or the latest version in the series of versions
+      # you are looking at
+      # ex: matched_versions = ["6.1.4.2", "6.1.4.1", "6.1.4"]
+      # if you have passed "6.1.4" and the list has the exact version, it will match and send
+      # the ruby version for it bu tif you had passed "6.1", then it will look for the
+      # latest version matching "6.1" which is "6.1.4.2" in this case and will return ruby
+      # version for it.
+      exact_version = matched_versions.include?(rv) ? rv: matched_versions[0]
```

Comment:
is this `rv: mathced_versions[0]` ok? I guess it should be `rv : matched_versions[0]`, isn't ruby complaining about an invalid syntax? (maybe it knows this is not `:rv => matched_versions[0]` but a ternary operator, but the missing `:` looks weird to me)